custom-connectors/CobbleStone - Contract Insight/apiDefinition.swagger.json

```diff
@@ -0,0 +1,974 @@
+{
+  "swagger": "2.0",
```

Comment:
@nizarhandal Please let us know the reason for adding this connector in "custom-connectors" folder. Please note that the connectors in the "custom-connectors" will not be published into the public regions.

If you are looking for certifying this connector, please make sure that you have registered your connector and place the connector inside "Certified-connectors" folder.

Reply:
Yes we want it to be published to public, so I moved it to certified connectors folder.
Thank you